contracts/FlowStakingCollection.cdc

```diff
@@ -749,8 +855,46 @@ pub contract FlowStakingCollection {
             return delegatorInfo
         }
 
+        /// Function to get the list of all of the machine accounts for a node in the collection
+        /// Key: Node ID
+        /// Value: Account address for the account that stores the QC or dkg object
+        pub fun getMachineAccounts(): {String: Address} {
+            let accountsRecord = FlowStakingCollection.account.copy<{String: Address}>(from: /storage/epochMachineAccounts)
```

Comment:
Should this path be a contract constant?

Reply:
It should be, but this is going to be an upgrade to the existing staking collection contract, so we can't add fields in an upgrade. Should I include the constant in the first deployment anyway, even though we won't use it until the upgrade?